app/controllers/catalog_controller.rb

```diff
@@ -57,8 +57,8 @@ class CatalogController < ApplicationController
     config.view.gallery.partials = [:index_header, :index]
     config.view.masonry.partials = [:index]
     config.view.slideshow.partials = [:index]
-    config.view.maps.type = 'placename_coord'
-    config.view.maps.placename_coord_field = 'placename_coords_ssim'
+    config.view.maps.type = 'bbox'
```

Comment:
I don't understand what this configuration line does or why it would necessarily change here.  Can you enlighten me?